src/test/librbd/test_WriteLogMap.cc

```diff
@@ -0,0 +1,318 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: move to `test/librbd/cache/rwl/...` -- also it's `test_LogMap` since that's the class being tested.

Reply:
Done